ietf/group/tests_appeals.py

```diff
@@ -0,0 +1,64 @@
+# Copyright The IETF Trust 2023, All Rights Reserved
+
+import debug    # pyflakes: ignore
+import datetime
+
+from pyquery import PyQuery
+
+from django.urls import reverse as urlreverse
+
+from ietf.utils.test_utils import login_testing_unauthorized, TestCase
+from ietf.group.factories import (
+    AppealFactory,
+    AppealArtifactFactory,
+)
+class AppealTests(TestCase):
+
+    def test_appeal_list_view(self):
+        appeal_date = datetime.date.today()-datetime.timedelta(days=14)
+        response_date = appeal_date+datetime.timedelta(days=8)
+        appeal = AppealFactory(name="A name to look for", date=appeal_date)
+        appeal_artifact = AppealArtifactFactory(appeal=appeal, artifact_type__slug="appeal", date=appeal_date)
+        response_artifact = AppealArtifactFactory(appeal=appeal, artifact_type__slug="response", content_type="application/pdf;charset=utf-8", date=response_date)
+
+        url = urlreverse("ietf.group.views.appeals", kwargs=dict(acronym="iab"))
+        r = self.client.get(url)
+        self.assertEqual(r.status_code, 200)
+        q = PyQuery(r.content)
+        self.assertEqual(len(q("#appeals > tbody > tr")), 1)
+        self.assertEqual(q("#appeal-1-date").text(), f"{appeal_date}")
+        self.assertEqual(f"{appeal_artifact.display_title()} - {appeal_date}", q("#artifact-1-1").text())
+        self.assertEqual(f"{response_artifact.display_title()} - {response_date}", q("#artifact-1-2").text())
+        self.assertIsNone(q("#artifact-1-1").attr("download"))
+        self.assertEqual(q("#artifact-1-2").attr("download"), response_artifact.download_name())
```

Comment:
It'd be nice to test somewhere that `download_name()` is behaving - I think all the tests so far would pass if it always returned "DoWnLoAd Me!" or (perhaps more plausibly) "some_prefix_.pdf".